dotCMS/src/main/java/com/dotmarketing/portlets/fileassets/business/FileAssetAPI.java

```diff
@@ -46,6 +47,10 @@
 
 	String TMP_UPLOAD = "tmp_upload";
 
+	public List<FileAsset> findFileAssetsByParentable(final Parentable parent,
```

Comment:
add doc

Reply:
instead of using many options, would be better to create an object with a builder pattern, so it can be extend on the future such as 

findFileAssetsByParentable(new FilterXXX.Builder().sortBy(yyy).includeWorking().includeArchived().user(user).respectFrontEndRoles(true).build());